### PR TITLE
ci: Show juju information

### PR DESCRIPTION
**Pull request by jneo8**

### CODE REVIEW

**Review by rgildein — CHANGES_REQUESTED** [review]

File: .github/workflows/_func.yaml
Comment (jneo8) [reply .github/workflows/_func.yaml]: Updated

Comment (jneo8) [reply .github/workflows/_func.yaml]: Hi @rgildein, can I have second review from you?

**Review by agileshaw — APPROVED** [review]

**Review by Pjack — APPROVED** [review]
LGTM

**Review by rgildein — APPROVED** [review]

**Pull request merged**
